src/online2bin/online2-tcp-nnet3-decode-faster.cc

```diff
@@ -425,6 +449,26 @@ Vector<BaseFloat> TcpServer::getChunk() {
   return buf;
 }
 
+void TcpServer::SaveRemainder(int remainder) {
```

Comment:
probably better to name the argument remainder_size.

Reply:
Sure